layouts/compliance/list.html

```diff
@@ -0,0 +1,55 @@
+{{ define "main" }}
+{{ $pagelist := .Pages }}
+{{ partial "hero.html" (dict "title" .Title) }}
+
+<div class="container mx-auto">
+  <section class="px-4 md:px-0">
+    <div class="container mx-auto max-w-4xl my-16 px-6">
+      <section class="mt-4 py-4 px-4 md:px-0">
+        <div class="container mx-auto text-center max-w-4xl">
+          <p>
+            Ensuring that your cloud infrastructure is compliant with frameworks like PCI-DSS, ISO 27001, and CIS
+            is critical for safeguarding sensitive data, maintaining customer trust, and adhering to legal and
+            regulatory requirements. Compliance with these frameworks demonstrates a commitment to implementing
+            best practices in security and risk management, which helps protect against data breaches,
+            cyber threats, and operational disruptions.
+          </p>
+        </div>
+      </section>
+      {{ $frameworks := slice "PCI DSS" "ISO 27001" "CIS" }}
+
+      {{ range $framework := $frameworks }}
+      {{ $pages := where $pagelist "Params.framework" $framework }}
+
+      {{ if $pages }}
+      {{ range $grouped := $pages.GroupByParam "cloud" }}
+      {{ range $grouped.Pages }}
+      {{ if eq .Params.cloud .Params.service }}
+      <section class="mt-4 py-4 px-4 md:px-0">
+        <div class="container mx-auto max-w-4xl">
+          <h3><a class="hover:text-blue-600" href="{{ .Permalink }}">{{ $framework }} Compliance for {{ .Params.cloud
+              }}</a>
+          </h3>
+          <p>
+            {{ .Params.description }}
+          </p>
+          <p>Learn more about configuring your AWS services to meet {{ $framework }} compliance:</p>
```

Comment:
Learn more about how Pulumi can help configure your AWS services to meet compliance...

I think we want to add in some stuff about Pulumi